frontend/src/component/ui/ChangeLogModal.tsx

```diff
@@ -25,6 +31,23 @@ export class ChangeLogModal extends Component<Props, State> {
     shouldShowFullChangeLog: false
   };
 
+  private modalRef = React.createRef<Modal>();
+
+  componentDidMount() {
+    if (this.props.shouldShowModal) {
+      this.open();
+    }
+  }
```

Comment:
```suggestion
```

Reply:
Should we really remove this, modal will not open if shouldShowModal prop is initially true

Reply:
@kapillamba4 We can call open function from its parent when ever we want.